### ISSUE TITLE
Premium features no longer working for self-hosters

### ISSUE THREAD

**Issue by twizzay-code** [YunoHost-Apps/snserver_ynh#129]
This is more of a news item. But I consider it a bug. Posting it here to provide answers to those wondering what is going on.

https://github.com/standardnotes/forum/issues/2427

SN now requires Self-hosters to pay $39/month to use "premium features" These premium features are already installed with the SNserver, but are not accessible without an "offline key".

Right now, SN is allowing self-hosters that were hosting before the change a lifetime license that will unlock premium features.

To obtain this, you need to email mo@standardnotes.com.

Related screenshots:
![01-31-2023_12:40%:18PM](https://user-images.githubusercontent.com/88040412/215901964-28282ac0-cb85-412a-9498-5b655f21721d.png)
![01-31-2023_11:40%:25AM](https://user-images.githubusercontent.com/88040412/215901956-af90148d-625b-45ad-a65c-937ea388a9a3.png)

Questions that are often asked:
1. Where are the files for the premium plan?
2. Is the offline subscription code reusable?
3. How is the code verified?

Response from Mo:
![01-31-2023_12:32%:32PM](https://user-images.githubusercontent.com/88040412/215901962-f7e9a8eb-3671-41e5-911d-3f37b61603b7.png)

**Comment by lucyferu** [comment]
that's very sad.

thanks for sharing the news.
i was actually a bit confused lol.

**Comment by drapado** [comment]
Self-hosters before the update can request a life key to enable the features forever. You can contact them as said in the discord server. I got mine easily

**Comment by lucyferu** [comment]
yep, i know, and i did the same, but this will change everything anyway.
i also think the people that were maintaining both the standard notes app for yunohost gave up maintaining it because, you know, that's not really the point.

i will keep using it for now because i need it for studying, but i am already looking at an alternative i can migrate to.

**Comment by webysther** [comment]
https://github.com/jackyzy823/standardnotes-app-images
